src/components/account/LoginAccountForm.tsx

```diff
@@ -147,8 +161,15 @@ const LoginAccountForm = () => {
         // });
       } catch (err) {
         console.error("Failed to onSubmit at LoginAccountForm:  ", err);
-        setAttempts((prev) => prev + 1);
-        if (attempts + 1 >= MAX_ATTEMPTS) {
+        const newCount = loginAttemptsStoreRef?.current?.count;
```

Comment:
Excessive optional chaining creates silent failures.

e.g., if `loginAttemptsStoreRef.current` is undefined: operations silently fail as `undefined >= MAX_ATTEMPTS` evaluates to false, preventing lockouts.

Undefined states require proper handling.